mantidproject/mslice
# Remove Group functionality

Removes the "Group" button on the GUI and all related code.

I have also take the opportunity to try and add a consistent layout to some of the files. This is a work in progress.

**To test:**

Start MSlice and observe that the Group functionality has gone. Try out some operations to ensure they still work.

Fixes #98
